tests/test_recruiters.py

```diff
@@ -21,10 +21,10 @@ def test_for_experiment(self, mod):
         assert mod.for_experiment(mock_exp) is mock_exp.recruiter
 
     def test_by_name_with_valid_name(self, mod):
-        assert mod.by_name('CLIRecruiter') == mod.CLIRecruiter
+        assert isinstance(mod.by_name('CLIRecruiter'), mod.CLIRecruiter)
```

Comment:
Ah, yes.. thanks.

Reply:
It used to return the class, but all the call sites actually wanted an instance, so I changed it to return an instance.